src/app/pages/partners/partner-details/partner-details.js

```diff
@@ -4,7 +4,7 @@ import Synopsis from './synopsis/synopsis';
 import Carousel from './carousel/carousel';
 import Reviews from './reviews/reviews';
 import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
-import TabGroup from '~/components/tab-group/tab-group.jsx';
+import TabGroup from '~/components/tab-group/tab-group';
```

Comment:
you can set a lint rule to catch this: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md